ts/components/ui/Markdown/rules/link.ts

```diff
@@ -1,17 +1,66 @@
+import { fromNullable, none, Option } from "fp-ts/lib/Option";
 import { Text } from "native-base";
 import * as React from "react";
 import { Linking } from "react-native";
+import { NavigationActions } from "react-navigation";
 import { ReactOutput, SingleASTNode, State } from "simple-markdown";
 
+import ROUTES from "../../../../navigation/routes";
 import { makeReactNativeRule } from "./utils";
 
+type ScreenNameToRouteMap = {
+  [key: string]: string;
+};
+
+// Prefix to match uri like `internal://screen/profile`
+const INTERNAL_TARGET_PREFIX = "internal://screen/";
```

Comment:
following @DenisFrezzato proposal, I think we should use the same prefix as deep links for consistency